src/main/java/ch/njol/skript/expressions/ExprFallenDistance.java

```diff
@@ -0,0 +1,96 @@
+/**
+ * This file is part of Skript.
+ *
+ * Skript is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Skript is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *
+ * Copyright 2011-2017 Peter Güttinger and contributors
+ */
+package ch.njol.skript.expressions;
+
+import org.bukkit.entity.Entity;
+import org.bukkit.event.Event;
+import org.eclipse.jdt.annotation.Nullable;
+
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.util.coll.CollectionUtils;
+
+@Name("Fallen distance")
+@Description({"The distance an entity has fallen for"})
+@Examples({"set all entities' fallen distance to 10",
+	"on damage:",
+	"\tsend \"%victim's fall distance%\" to victim"})
+@Since("INSERT VERSION")
+public class ExprFallenDistance extends SimplePropertyExpression<Entity, Number> {
+	
+	static {
+		register(ExprFallenDistance.class, Number.class, "[the] (fall[en]|fell[ed]) (distance|height)", "entities");
+	}
+	
+	@Nullable
+	@Override
+	public Number convert(Entity entity) {
+		return entity.getFallDistance();
+	}
+	
+	@Nullable
+	@Override
+	public Class<?>[] acceptChange(ChangeMode mode) {
+		return (mode == ChangeMode.RESET || mode == ChangeMode.REMOVE_ALL ||mode == ChangeMode.DELETE) ? null : CollectionUtils.array(Number.class);
```

Comment:
missing a space after the second `||` - if you're using intellij and it's git integration it might be worthwhile to enable the `reformat code before commit` option to catch small things like this

Reply:
The issue with that is that it would always format it and if it hasn't been formatted beforehand I'm told to format it in another pull request

Reply:
I mean, you can avoid committing changes unrelated to the PR or do them on a separate branch if needed